lib/BulkRenewalDialog/BulkRenewInfo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import Button from '@folio/stripes-components/lib/Button';
+import Icon from '@folio/stripes-components/lib/Icon';
+import Layout from '@folio/stripes-components/lib/Layout';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+import BulkRenewedLoansList from './BulkRenewedLoansList';
+
+class BulkRenewInfo extends React.Component {
+  static propTypes = {
+    stripes: PropTypes.object,
+    onCancel: PropTypes.func,
+    successRenewals: PropTypes.arrayOf(PropTypes.object),
+    failedRenewals: PropTypes.arrayOf(PropTypes.object),
+    loanPolicies: PropTypes.object,
+    requestCounts: PropTypes.object,
+    errorMessages: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props);
+    this.connectedLoanList = props.stripes.connect(BulkRenewedLoansList);
+  }
+
+  render() {
+    const { successRenewals, failedRenewals } = this.props;
+    const iconAlignStyle = { display: 'flex', alignItems: 'center' };
```

Comment:
Since you're already importing `Layout`, could you use the `flex` style it has to avoid inlining styles?

Reply:
@doytch oh! was there one? missed it then. Will do!